ui-library-filesharing-chat-composite/app/src/App.tsx

```diff
@@ -0,0 +1,138 @@
+import { AzureCommunicationTokenCredential } from '@azure/communication-common';
+import {
+  ChatComposite,
+  ChatAdapter,
+  createAzureCommunicationChatAdapter,
+  FileUploadHandler,
+  FileUploadManager,
+  FileDownloadHandler
+} from '@azure/communication-react';
+import React, { useEffect, useMemo, useState } from 'react';
+import axios from 'axios';
+import FormData from 'form-data';
+import { v4 } from 'uuid';
+
+const MAX_FILE_SIZE_MB = 50 * 1024 * 1024;
+const UNSUPPORTED_FILES = ['exe', 'bat', 'dat'];
+
+function App(): JSX.Element {
+  // Common variables
+  const endpointUrl = '<Azure Communication Services Resource Endpoint>';
+  const token = '<Azure Communication Services Resource Access Token>';
+  const userId = '<User Id associated to the token>';
+  const threadId = '<Get thread id from chat service>';
+  const displayName = '<Display Name>';
+  
+  const [chatAdapter, setChatAdapter] = useState<ChatAdapter>();
+
+  // We can't even initialize the Chat and Call adapters without a well-formed token.
+  const credential = useMemo(() => {
+    try {
+      return new AzureCommunicationTokenCredential(token);
+    } catch {
+      console.error('Failed to construct token credential');
+      return undefined;
+    }
+  }, [token]);
+
+  useEffect(() => {
```

Comment:
Please use the new hooks here instead: https://azure.github.io/communication-ui-library/?path=/docs/quickstarts-composites--page#initialize-composites